### PR TITLE
feat: switch python client to use openapi-generator

### PR DESCRIPTION
**Pull request by tomplus**
The Python will be generated by openapi-generator. I also prepared this PR: https://github.com/kubernetes-client/python/pull/738 to show what changes will be introduced to the generated client. Unfortunately this client used very old version of swagger-codegen.

Ref: https://github.com/kubernetes-client/gen/issues/93

PTAL: @yliaog @roycaihw @mbohlool @brendandburns

### CODE REVIEW

File: openapi/python.sh [hunk 1]
```diff
@@ -43,29 +43,23 @@ pushd "${OUTPUT_DIR}" > /dev/null
...
-# TODO: Remove this when above merges
-patch "${OUTPUT_DIR}/${PACKAGE_NAME}/api_client.py" "${SCRIPT_ROOT}/python-api_client.py.patch"
-
+# Post-processing of the generated Python wrapper.
 find "${OUTPUT_DIR}/test" -type f -name \*.py -exec sed -i 's/\bclient/kubernetes.client/g' {} +
 find "${OUTPUT_DIR}" -path "${OUTPUT_DIR}/base" -prune -o -type f -a -name \*.md -exec sed -i 's/\bclient/kubernetes.client/g' {} +
 find "${OUTPUT_DIR}" -path "${OUTPUT_DIR}/base" -prune -o -type f -a -name \*.md -exec sed -i 's/kubernetes.client-python/client-python/g' {} +
+
+# fix imports
+find "${OUTPUT_DIR}/client/" -type f -name \*.py -exec sed -i 's/import client\./import kubernetes.client./g' {} +
```
Comment (tomplus) [inline openapi/python.sh]: Absolute vs relative imports. It's not related to the openapi-generator too, the latest version of swagger-codegen uses absolute imports and here it's fixed by adding prefix `kubernetes.`.

**Comment by tomplus** [comment]
@yliaog @roycaihw please take a look, you can see generated code here: https://github.com/kubernetes-client/python/pull/738

**Comment by brendandburns** [comment]
@tomplus I'm happy to merge this if/when it is ready to go, but I defer to other reviewers...

**Comment by roycaihw** [comment]
Thanks for putting this together! I looked at https://github.com/kubernetes-client/python/pull/738 and it's pretty neat. I had some minor questions that aren't blocking, so

/lgtm

I'd like to try this for the next 1.15 alpha release :)

**Comment by roycaihw** [comment]
/assign @brendandburns 

I tried picking this locally and the generated results look good. Could you approve this pull so we can generate using the upstream `gen` repo?

**Comment by yliaog** [comment]
@brendandburns, could you add @roycaihw and @yliaog to the admin group of this repo, i think it is https://github.com/orgs/kubernetes-client/teams/gen-admins

**Comment by brendandburns** [comment]
Sorry for the delay, please ping me on slack or email in the future.

Happy to had you to the admins.

/approve

**Pull request merged**